assets/dev/js/editor/utils/controls-css-parser.js

```diff
@@ -262,6 +262,14 @@ ControlsCSSParser = elementorModules.ViewModule.extend( {
 			propertyName = propertyName.replace( '_', '-' );
 
 			value = `var( --e-global-${ control.groupType }-${ id }-${ propertyName } )`;
+
+			if ( control.groupPrefix + 'font_family' === control.name ) {
+				const defaultFonts = control.selector_value.match( /VALUE}}"(.*)/ );
```

Comment:
Expose the kit `default_generic_fonts` to the config and use it.

Reply:
Since we already have the information, the purpose is to not initialize the kit settings if it's not necessary